docs/fsharp/style-guide/formatting.md

```diff
@@ -196,6 +196,21 @@ F# inherits both the postfix ML style of naming generic types (for example, `int
 
 For all other types, use the prefix form.
 
+## Formatting tuples
+
+A tuple instantiation should be parenthesized, and the delimiting commas within should be followed by a single space, for example: `(1, 2)`, `(x, y, z)`.
+
+A commonly accepted exception is to omit parentheses in pattern matching of tuples:
```

Comment:
"A commonly accepted exception is to" ---> "It is commonly accepted to"

Referencing this as an exception to the rule is logical, but the phrase is a bit awkward to read, and people will understand that this is a deviation from the norm in this one instance.

Reply:
Note that I did not change this section at all, just moved it slightly to have the declaration and instance usage of records near each other

Reply:
But when we already here I can change it. One other thing I noticed, Isn't

```fsharp
let x, y = z // Destructuring
```

perfectly fine as well? We might as well add this to the sample.